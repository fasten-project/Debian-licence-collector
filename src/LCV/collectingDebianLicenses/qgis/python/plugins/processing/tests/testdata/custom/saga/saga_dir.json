{
    "content": [
        {
            "hidden": false,
            "name": "points_regression.cpg",
            "stat": {
                "perms": "rw-r--r--",
                "size": 5,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "points_regression.dbf",
            "stat": {
                "perms": "rw-r--r--",
                "size": 290,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "points_regression.prj",
            "stat": {
                "perms": "rw-r--r--",
                "size": 143,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "points_regression.qpj",
            "stat": {
                "perms": "rw-r--r--",
                "size": 257,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "points_regression.shp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 268,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "points_regression.shx",
            "stat": {
                "perms": "rw-r--r--",
                "size": 148,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "training_areas.cpg",
            "stat": {
                "perms": "rw-r--r--",
                "size": 5,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "training_areas.dbf",
            "stat": {
                "perms": "rw-r--r--",
                "size": 161,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "training_areas.prj",
            "stat": {
                "perms": "rw-r--r--",
                "size": 143,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "training_areas.qpj",
            "stat": {
                "perms": "rw-r--r--",
                "size": 257,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "training_areas.shp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 508,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "training_areas.shx",
            "stat": {
                "perms": "rw-r--r--",
                "size": 124,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        }
    ],
    "directory": "saga",
    "package": "qgis",
    "path": "qgis/3.16.15+dfsg-1/python/plugins/processing/tests/testdata/custom/saga",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 0,
        "license": "/copyright/license/qgis/3.16.15+dfsg-1/",
        "metric": {
            "size": 814172
        },
        "pts_link": "https://tracker.debian.org/pkg/qgis",
        "sloc": [
            [
                "cpp",
                1144458
            ],
            [
                "python",
                173601
            ],
            [
                "xml",
                13652
            ],
            [
                "perl",
                3382
            ],
            [
                "sh",
                3088
            ],
            [
                "sql",
                2446
            ],
            [
                "ansic",
                2199
            ],
            [
                "yacc",
                1021
            ],
            [
                "lex",
                570
            ],
            [
                "javascript",
                505
            ],
            [
                "lisp",
                411
            ],
            [
                "makefile",
                233
            ]
        ],
        "suites": [
            "bookworm",
            "sid"
        ],
        "vcs_browser": "https://salsa.debian.org/debian-gis-team/qgis",
        "vcs_type": "git"
    },
    "type": "directory",
    "version": "3.16.15+dfsg-1"
}