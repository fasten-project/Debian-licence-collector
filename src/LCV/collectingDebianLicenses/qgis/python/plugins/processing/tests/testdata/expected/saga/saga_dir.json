{
    "content": [
        {
            "hidden": false,
            "name": "add_grid_to_points.cpg",
            "stat": {
                "perms": "rw-r--r--",
                "size": 5,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "add_grid_to_points.dbf",
            "stat": {
                "perms": "rw-r--r--",
                "size": 314,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "add_grid_to_points.mshp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 1074,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "add_grid_to_points.prj",
            "stat": {
                "perms": "rw-r--r--",
                "size": 143,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "add_grid_to_points.qpj",
            "stat": {
                "perms": "rw-r--r--",
                "size": 257,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "add_grid_to_points.shp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 268,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "add_grid_to_points.shx",
            "stat": {
                "perms": "rw-r--r--",
                "size": 148,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "add_grid_to_shapes.dbf",
            "stat": {
                "perms": "rw-r--r--",
                "size": 255,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "add_grid_to_shapes.mshp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 1105,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "add_grid_to_shapes.prj",
            "stat": {
                "perms": "rw-r--r--",
                "size": 256,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "add_grid_to_shapes.shp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 364,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "add_grid_to_shapes.shx",
            "stat": {
                "perms": "rw-r--r--",
                "size": 124,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "buffer_polys.dbf",
            "stat": {
                "perms": "rw-r--r--",
                "size": 401,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "buffer_polys.mshp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 1601,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "buffer_polys.prj",
            "stat": {
                "perms": "rw-r--r--",
                "size": 256,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "buffer_polys.shp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 1280,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "buffer_polys.shx",
            "stat": {
                "perms": "rw-r--r--",
                "size": 140,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "cross_profiles.dbf",
            "stat": {
                "perms": "rw-r--r--",
                "size": 4273,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "cross_profiles.mshp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 2436,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "cross_profiles.prj",
            "stat": {
                "perms": "rw-r--r--",
                "size": 256,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "cross_profiles.shp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 1508,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "cross_profiles.shx",
            "stat": {
                "perms": "rw-r--r--",
                "size": 228,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "merge_vector_layers.dbf",
            "stat": {
                "perms": "rw-r--r--",
                "size": 1393,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "merge_vector_layers.mshp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 1520,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "merge_vector_layers.prj",
            "stat": {
                "perms": "rw-r--r--",
                "size": 256,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "merge_vector_layers.shp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 716,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "merge_vector_layers.shx",
            "stat": {
                "perms": "rw-r--r--",
                "size": 276,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "multiple_regression_points_grids_coef.dbf",
            "stat": {
                "perms": "rw-r--r--",
                "size": 708,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "multiple_regression_points_grids_coef.mtab",
            "stat": {
                "perms": "rw-r--r--",
                "size": 2698,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "multiple_regression_points_grids_coef.prj",
            "stat": {
                "perms": "rw-r--r--",
                "size": 256,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "multiple_regression_points_grids_model.dbf",
            "stat": {
                "perms": "rw-r--r--",
                "size": 607,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "multiple_regression_points_grids_model.mtab",
            "stat": {
                "perms": "rw-r--r--",
                "size": 2349,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "multiple_regression_points_grids_model.prj",
            "stat": {
                "perms": "rw-r--r--",
                "size": 256,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "multiple_regression_points_grids_res.dbf",
            "stat": {
                "perms": "rw-r--r--",
                "size": 477,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "multiple_regression_points_grids_res.mshp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 2288,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "multiple_regression_points_grids_res.prj",
            "stat": {
                "perms": "rw-r--r--",
                "size": 256,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "multiple_regression_points_grids_res.shp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 268,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "multiple_regression_points_grids_res.shx",
            "stat": {
                "perms": "rw-r--r--",
                "size": 148,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "multiple_regression_points_grids_steps.dbf",
            "stat": {
                "perms": "rw-r--r--",
                "size": 818,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "multiple_regression_points_grids_steps.mtab",
            "stat": {
                "perms": "rw-r--r--",
                "size": 2932,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "multiple_regression_points_grids_steps.prj",
            "stat": {
                "perms": "rw-r--r--",
                "size": 256,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "point_distances.dbf",
            "stat": {
                "perms": "rw-r--r--",
                "size": 2721,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "point_distances.mtab",
            "stat": {
                "perms": "rw-r--r--",
                "size": 1411,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "point_distances.prj",
            "stat": {
                "perms": "rw-r--r--",
                "size": 256,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "polygon_difference.dbf",
            "stat": {
                "perms": "rw-r--r--",
                "size": 83,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "polygon_difference.mshp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 1409,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "polygon_difference.prj",
            "stat": {
                "perms": "rw-r--r--",
                "size": 676,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "polygon_difference.shp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 948,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "polygon_difference.shx",
            "stat": {
                "perms": "rw-r--r--",
                "size": 148,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "polygon_identity.dbf",
            "stat": {
                "perms": "rw-r--r--",
                "size": 147,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "polygon_identity.mshp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 1445,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "polygon_identity.prj",
            "stat": {
                "perms": "rw-r--r--",
                "size": 676,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "polygon_identity.shp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 1492,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "polygon_identity.shx",
            "stat": {
                "perms": "rw-r--r--",
                "size": 180,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "polygon_intersect.dbf",
            "stat": {
                "perms": "rw-r--r--",
                "size": 117,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "polygon_intersect.mshp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 1448,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "polygon_intersect.prj",
            "stat": {
                "perms": "rw-r--r--",
                "size": 676,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "polygon_intersect.shp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 644,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "polygon_intersect.shx",
            "stat": {
                "perms": "rw-r--r--",
                "size": 132,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "polygon_self_intersection.dbf",
            "stat": {
                "perms": "rw-r--r--",
                "size": 115,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "polygon_self_intersection.mshp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 1342,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "polygon_self_intersection.prj",
            "stat": {
                "perms": "rw-r--r--",
                "size": 676,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "polygon_self_intersection.shp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 508,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "polygon_self_intersection.shx",
            "stat": {
                "perms": "rw-r--r--",
                "size": 124,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "polygon_symmetrical_difference.dbf",
            "stat": {
                "perms": "rw-r--r--",
                "size": 157,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "polygon_symmetrical_difference.mshp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 1487,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "polygon_symmetrical_difference.prj",
            "stat": {
                "perms": "rw-r--r--",
                "size": 676,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "polygon_symmetrical_difference.shp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 1796,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "polygon_symmetrical_difference.shx",
            "stat": {
                "perms": "rw-r--r--",
                "size": 196,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "polygon_union.dbf",
            "stat": {
                "perms": "rw-r--r--",
                "size": 177,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "polygon_union.mshp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 1436,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "polygon_union.prj",
            "stat": {
                "perms": "rw-r--r--",
                "size": 676,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "polygon_union.shp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 2340,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "polygon_union.shx",
            "stat": {
                "perms": "rw-r--r--",
                "size": 228,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "polygon_update.dbf",
            "stat": {
                "perms": "rw-r--r--",
                "size": 95,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "polygon_update.mshp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 1397,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "polygon_update.prj",
            "stat": {
                "perms": "rw-r--r--",
                "size": 676,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "polygon_update.shp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 1524,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "polygon_update.shx",
            "stat": {
                "perms": "rw-r--r--",
                "size": 180,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "snap_points_to_lines_moves.dbf",
            "stat": {
                "perms": "rw-r--r--",
                "size": 101,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "snap_points_to_lines_moves.mshp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 1220,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "snap_points_to_lines_moves.prj",
            "stat": {
                "perms": "rw-r--r--",
                "size": 256,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "snap_points_to_lines_moves.shp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 452,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "snap_points_to_lines_moves.shx",
            "stat": {
                "perms": "rw-r--r--",
                "size": 132,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "snap_points_to_lines_result.dbf",
            "stat": {
                "perms": "rw-r--r--",
                "size": 146,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "snap_points_to_lines_result.mshp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 1220,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "snap_points_to_lines_result.prj",
            "stat": {
                "perms": "rw-r--r--",
                "size": 256,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "snap_points_to_lines_result.shp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 352,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "snap_points_to_lines_result.shx",
            "stat": {
                "perms": "rw-r--r--",
                "size": 172,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "snap_points_to_points_moves.dbf",
            "stat": {
                "perms": "rw-r--r--",
                "size": 137,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "snap_points_to_points_moves.mshp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 1221,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "snap_points_to_points_moves.prj",
            "stat": {
                "perms": "rw-r--r--",
                "size": 256,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "snap_points_to_points_moves.shp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 804,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "snap_points_to_points_moves.shx",
            "stat": {
                "perms": "rw-r--r--",
                "size": 164,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "snap_points_to_points_result.dbf",
            "stat": {
                "perms": "rw-r--r--",
                "size": 146,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "snap_points_to_points_result.mshp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 1221,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "snap_points_to_points_result.prj",
            "stat": {
                "perms": "rw-r--r--",
                "size": 256,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "snap_points_to_points_result.shp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 352,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "snap_points_to_points_result.shx",
            "stat": {
                "perms": "rw-r--r--",
                "size": 172,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "watershed_segmentation_borders.mgrd",
            "stat": {
                "perms": "rw-r--r--",
                "size": 2456,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "watershed_segmentation_borders.prj",
            "stat": {
                "perms": "rw-r--r--",
                "size": 206,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "watershed_segmentation_borders.sdat",
            "stat": {
                "perms": "rw-r--r--",
                "size": 132000,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "watershed_segmentation_borders.sdat.aux.xml",
            "stat": {
                "perms": "rw-r--r--",
                "size": 245,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "watershed_segmentation_borders.sgrd",
            "stat": {
                "perms": "rw-r--r--",
                "size": 304,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "watershed_segmentation_seeds.dbf",
            "stat": {
                "perms": "rw-r--r--",
                "size": 35809,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "watershed_segmentation_seeds.mshp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 2593,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "watershed_segmentation_seeds.prj",
            "stat": {
                "perms": "rw-r--r--",
                "size": 206,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "watershed_segmentation_seeds.shp",
            "stat": {
                "perms": "rw-r--r--",
                "size": 11972,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "watershed_segmentation_seeds.shx",
            "stat": {
                "perms": "rw-r--r--",
                "size": 3492,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "watershed_segmentation_segments.mgrd",
            "stat": {
                "perms": "rw-r--r--",
                "size": 2474,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "watershed_segmentation_segments.prj",
            "stat": {
                "perms": "rw-r--r--",
                "size": 206,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "watershed_segmentation_segments.sdat",
            "stat": {
                "perms": "rw-r--r--",
                "size": 522200,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "watershed_segmentation_segments.sdat.aux.xml",
            "stat": {
                "perms": "rw-r--r--",
                "size": 245,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "watershed_segmentation_segments.sgrd",
            "stat": {
                "perms": "rw-r--r--",
                "size": 316,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "zonal_grid_stats.dbf",
            "stat": {
                "perms": "rw-r--r--",
                "size": 163,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "zonal_grid_stats.mtab",
            "stat": {
                "perms": "rw-r--r--",
                "size": 2146,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        }
    ],
    "directory": "saga",
    "package": "qgis",
    "path": "qgis/3.16.15+dfsg-1/python/plugins/processing/tests/testdata/expected/saga",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 0,
        "license": "/copyright/license/qgis/3.16.15+dfsg-1/",
        "metric": {
            "size": 814172
        },
        "pts_link": "https://tracker.debian.org/pkg/qgis",
        "sloc": [
            [
                "cpp",
                1144458
            ],
            [
                "python",
                173601
            ],
            [
                "xml",
                13652
            ],
            [
                "perl",
                3382
            ],
            [
                "sh",
                3088
            ],
            [
                "sql",
                2446
            ],
            [
                "ansic",
                2199
            ],
            [
                "yacc",
                1021
            ],
            [
                "lex",
                570
            ],
            [
                "javascript",
                505
            ],
            [
                "lisp",
                411
            ],
            [
                "makefile",
                233
            ]
        ],
        "suites": [
            "bookworm",
            "sid"
        ],
        "vcs_browser": "https://salsa.debian.org/debian-gis-team/qgis",
        "vcs_type": "git"
    },
    "type": "directory",
    "version": "3.16.15+dfsg-1"
}